### ISSUE TITLE
Most assets in Asset Bundles with ExplicitDataLayout set to 0 aren't given the proper container path

### ISSUE THREAD

**Issue by LudwigvonWu** [Perfare/AssetStudio#567]
Some Asset Bundles don't specify a container path for every asset, but rather set ExplicitDataLayout to 0 and then in the container map they specify the container path of the first asset and then how many assets share the same path through the preloadSize AssetInfo.

Currently, AssetStudio doesn't properly parse said information and thus only assign the container path to the first asset, leaving the following assets without any container path and thus preventing correct export through the "Group exported assets by container path" option.

**Comment by Perfare** [comment]
Fixed.

**Comment by LudwigvonWu** [comment]
Sorry to reopen the issue, but it looks like I found another corner case in which the assets aren't given a specified path: in some bundles the m_PreloadTable is empty and as such preloadIndex and preloadSize are set to 0 in the m_Container map, even though the container path is still specified. So far I noticed this in the bundles with m_PathFlags set to 7. My best guess is that in this case all the assets in the bundle are still meant to be given the specified container path.
